sklearn/feature_extraction/heterogeneous.py

```diff
@@ -0,0 +1,148 @@
+from scipy import sparse
+import numpy as np
+
+from ..base import BaseEstimator, TransformerMixin
+from ..pipeline import _fit_one_transformer, _fit_transform_one, _transform_one
+from ..externals.joblib import Parallel, delayed
+from ..externals.six import iteritems
+
+
+class ColumnTransformer(BaseEstimator, TransformerMixin):
+    """Applies transformers to columns of a dataframe / dict.
```

Comment:
While I see of point of this transformer on dataframe and dicts, I find it too bad we cannot apply it on Numpy arrays. I would love to have see a built-in to apply transformers on selected columns only.